### PR TITLE
pyccl.correlations.correlation doc

### PR DESCRIPTION
**Pull request by jecampagne**
I take some piece of text from the section 2.4.2. of arXiv:1812.05995v2 CCL article.

### CODE REVIEW

**Comment by jecampagne** [comment]
This PR is related to issue  #871

File: pyccl/correlations.py [hunk 1]
```diff
@@ -30,6 +30,33 @@
 def correlation(cosmo, ell, C_ell, theta, type='NN', corr_type=None,
                 method='fftlog'):
     """Compute the angular correlation function.
+    
+    
+
```
Comment (nikfilippas) [inline pyccl/correlations.py:35]: just 1 empty line, not 3

File: pyccl/correlations.py [hunk 2]
```diff
@@ -30,6 +30,33 @@
...
+    .. math::
+        \\xi^{ab}_\\pm(\\theta) = \\sum_\\ell\\frac{2\\ell+1}{4\\pi}\,(\\pm1)^{s_b}\\,
+                         C^{ab\\pm}_\\ell\\,d^\\ell_{s_a,\\pm s_b}(\\theta)
+    
+
```
Comment (nikfilippas) [inline pyccl/correlations.py:40]: just 1 empty line, not 2

File: pyccl/correlations.py [hunk 3]
```diff
@@ -30,6 +30,33 @@
...
+    where :math:`\\theta` is the angle between the two fields :math:`a` and :math:`b` with spins
+    :math:`s_a` and :math:`s_b` after alignement of their tangential coordinate.  :math:`d^\\ell_{mm'}`
+    are the Wigner-d matrices and we have defined the power spectra
+
+
```
Comment (nikfilippas) [inline pyccl/correlations.py:45]: just 1 empty line, not 2

File: pyccl/correlations.py [hunk 4]
```diff
@@ -30,6 +30,33 @@
...
+    .. math::
+       C^{ab\\pm}_\\ell\\equiv\\left(C^{a_Eb_E}_\\ell\\pm C^{a_Bb_B}_\\ell\\right)
+                  +i\\left(C^{a_Bb_E}_\\ell\\mp C^{a_Eb_B}_\\ell\\right)
+
+
```
Comment (nikfilippas) [inline pyccl/correlations.py:50]: just 1 empty line, not 2

**Comment by nikfilippas** [comment]
Closing this as #980 is already a fork of this branch, and the changes there reflect the changes here too.

**Pull request closed without merging**